src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <DemoWalkthroughPageLayout :drawArrows="drawArrows">
+    <h1 class="heading">
+      Shoppers
+    </h1>
+
+    <p>
+      To showcase how Amazon Personalize tailors product recommendations based on a user’s historical and real time
+      click-through behavior, the Retail Demo Store includes several fictitious shoppers, each with different shopping
+      preferences.
+    </p>
+
+    <div :class="{ 'images mt-5': true, 'images--mobile': isMobile }">
+      <div ref="switch-shoppers-text" class="switch-shoppers-text">
+        Switch shoppers by clicking on the shopper details located next to the shopping cart icon. Top right corner.
+      </div>
+
+      <div class="switch-shoppers">
+        <img
+          :src="isMobile ? '/switch-shoppers-mobile.png' : '/switch-shoppers-desktop.png'"
+          alt=""
+          class="img-fluid"
+          ref="switch-shoppers-img"
+        />
+      </div>
+
+      <div class="select-shoppers-text">
+        <p>
+          This demo allows you to select shoppers from the store users’ dataset, based on shopping preferences, age and
+          gender.
```

Comment:
To be inclusive, the gender will  not be something that can be selected

Reply:
Updated